keps/sig-auth/5018-dra-adminaccess/kep.yaml

```diff
@@ -0,0 +1,46 @@
+title: DRA Admin Access
+kep-number: 5018
+authors:
+  - "@ritazh"
+owning-sig: sig-auth
+status: implementable
+creation-date: 2025-01-02
+reviewers:
+  - "@pohly"
+  - "@liggitt"
+  - "@enj"
+approvers:
+  - "@enj"
+  -  # SIG-Node
```

Comment:
@SergeyKanzhelev, @mrunalp: can one of you approve for SIG Node?

Reply:
You can add me here.